packages/@aws-cdk/aws-codeguruprofiler/test/integ.profiler-group.ts

```diff
@@ -0,0 +1,30 @@
+import { AccountRootPrincipal, Role } from '@aws-cdk/aws-iam';
+import { App, Stack, StackProps } from '@aws-cdk/core';
+import { ProfilingGroup } from '../lib/profiling-group';
+
+class ProfilerGroupIntegrationTest extends Stack {
+  constructor(scope: App, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    const profilingGroup = new ProfilingGroup(this, 'MyProfilingGroup', {
+      profilingGroupName: 'MyAwesomeProfilingGroup',
+    });
```

Comment:
Check that the default works:

```suggestion
    const profilingGroup = new ProfilingGroup(this, 'MyProfilingGroup');
```